apps/ledger-live-mobile/src/newArch/components/QueuedDrawer/TestScreens.tsx

```diff
@@ -181,11 +181,18 @@ export const MainTestScreen = () => {
   const [drawer3RequestingToBeOpened, setDrawer3RequestingToBeOpened] = useState(false);
   const [drawer4ForcingToBeOpened, setDrawer4ForcingToBeOpened] = useState(false);
 
-  const handleDrawer1Close = useCallback(() => setDrawer1RequestingToBeOpened(false), []);
+  const handleDrawer1Close = () => setDrawer1RequestingToBeOpened(false); // purposely not using useCallback to check if drawer behaves well when onClose prop changes
   const handleDrawer2Close = useCallback(() => setDrawer2RequestingToBeOpened(false), []);
   const handleDrawer3Close = useCallback(() => setDrawer3RequestingToBeOpened(false), []);
   const handleDrawer4Close = useCallback(() => setDrawer4ForcingToBeOpened(false), []);
 
+  // adding this to force a re-render every 200ms and ensure the drawers behave well when rerendering
+  const [_, setNonce] = useState(0);
+  useEffect(() => {
+    const interval = setInterval(() => setNonce(nonce => nonce + 1), 200);
+    return () => clearInterval(interval);
+  }, []);
```

Comment:
This feels really hacky and if we need this I'd say we have a bigger problem somewhere else

Reply:
I'll rewrite the comment.

Reply:
Would this be clearer @Justkant ?
`adding this to force a re-render every 200ms and ensure that we test that the drawers behave well when rerendering`

Reply:
Actually I don't need this artificial way of re-rendering to test that scenario since the state of TestScreens already changes when requesting to open a new drawer, and it triggers the bug if I revert my fix.
I'll remove this part.

Reply:
Oh my bad, I didn't see it was only for tests purposes